data/txDupCache_test.go

```diff
@@ -371,3 +371,132 @@ func benchmarkDigestCache(b *testing.B, m cacheMaker, numThreads int) {
 	}
 	wg.Wait()
 }
+
+// TestTxHandlerSaltedCacheValues checks values are stored correctly
+func TestTxHandlerSaltedCacheValues(t *testing.T) {
+	partitiontest.PartitionTest(t)
+	t.Parallel()
+
+	const size = 2
+	cache := makeSaltedCache(size)
+	cache.Start(context.Background(), 0)
+	require.Zero(t, cache.Len())
+
+	type snd struct {
+		id int
+	}
+
+	d, v, p, found := cache.innerCheck([]byte{1})
+	require.False(t, found)
+	require.Nil(t, p)
+	require.Nil(t, v)
+	require.NotNil(t, d)
+	require.NotEmpty(t, d)
```

Comment:
This is just to confirm the data structure is empty before the test starts, or is there other utility? A brief comment might help. If you're just looking to confirm it's empty, why not use your Len function?